### PR TITLE
[Nim] Bootstrap track

### LINKED ISSUE #1562: [Nim] Please bootstrap Nim track
We have mostly completed the test-runner.
Looking forward to contributing to v3!!!!

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes #1562

### CODE REVIEW

File: languages/nim/reference/implementing-a-concept-exercise.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# How to implement an Nim concept exercise
```
Comment (ynfle) [inline languages/nim/reference/implementing-a-concept-exercise.md:1]:
```suggestion
# How to implement a Nim concept exercise
```

**Comment by ynfle** [comment]
Thanks!

**Comment by ErikSchierboom** [comment]
Merged @ynfle !

**Comment by ynfle** [comment]
Thank you

**Pull request merged**